src/sentry/static/sentry/app/views/eventsV2/table/tableActions.tsx

```diff
@@ -93,98 +102,71 @@ renderAsyncExportButton.propTypes = {
 function renderEditButton(canEdit: boolean, props: Props) {
   const onClick = canEdit ? props.onEdit : undefined;
   return (
-    <HeaderButton disabled={!canEdit} onClick={onClick} data-test-id="grid-edit-enable">
-      <IconEdit size="xs" />
-      {t('Edit Columns')}
-    </HeaderButton>
+    <Button
+      size="small"
+      disabled={!canEdit}
+      onClick={onClick}
+      data-test-id="grid-edit-enable"
+      icon={<IconStack size="xs" />}
+    >
+      {t('Columns')}
+    </Button>
   );
 }
 // Placate eslint proptype checking
 renderEditButton.propTypes = {
   onEdit: PropTypes.func,
 };
 
-function HeaderActions(props: Props) {
-  const noEditMessage = t('Requires discover query feature.');
-  const editFeatures = ['organizations:discover-query'];
-
-  const renderDisabled = p => (
-    <Hovercard
-      body={
-        <FeatureDisabled
-          features={p.features}
-          hideHelpToggle
-          message={noEditMessage}
-          featureName={noEditMessage}
-        />
-      }
-    >
-      {p.children(p)}
-    </Hovercard>
-  );
-  return (
-    <Feature
-      hookName="feature-disabled:grid-editable-actions"
-      renderDisabled={renderDisabled}
-      features={editFeatures}
-    >
-      {({hasFeature}) => (
-        <React.Fragment>
-          {renderEditButton(hasFeature, props)}
-          {renderDownloadButton(hasFeature, props)}
-        </React.Fragment>
-      )}
-    </Feature>
-  );
-}
-
-const HeaderButton = styled('button')<{disabled?: boolean}>`
-  display: flex;
-  align-items: center;
-
-  background: none;
-  border: none;
-  color: ${p => (p.disabled ? p.theme.gray400 : p.theme.gray600)};
-  cursor: ${p => (p.disabled ? 'default' : 'pointer')};
-  font-size: ${p => p.theme.fontSizeSmall};
+class HeaderActions extends React.Component<Props, State> {
+  state = {showTags: true};
```

Comment:
Could this allow for state to differ between the parent components and this one? For example if the localstorage value is set to 0, the parent component would think that that the sidebar is closed, while this component would have the state as open.